sql/database.sql

```diff
@@ -10765,3 +10765,77 @@ CREATE TABLE `patient_birthday_alert` (
   `turned_off_on` date NOT NULL,
   PRIMARY KEY  (`pid`,`user_id`)
 ) ENGINE=InnoDB;
+
+-----------------------------------------------------------
+--
+-- Table structure for table `morphine_eq`
+--
+
+DROP TABLE IF EXISTS `morphine_eq`;
+CREATE TABLE IF NOT EXISTS `morphine_eq` (
+  `id` int(11) NOT NULL AUTO_INCREMENT,
+  `datetime` datetime NOT NULL,
+  `encounter` int(10) NOT NULL,
+  `last` int(6) NOT NULL,
+  `next` int(6) NOT NULL,
+  `notes` varchar(90) NOT NULL,
+  `pid` int(6) NOT NULL,
```

Comment:
generally good idea to index the pid also (also, is this big enough for a pid?)

Reply:
Increased the size of the PID to 11.